test/integration/init.ts

```diff
@@ -1,32 +1,30 @@
 import { ethers } from "ethers";
-import Web3 from "web3";
+import { OpenSeaSDK } from "../../src/sdk";
+import { Network } from "../../src/types";
 import {
   ALCHEMY_API_KEY,
   MAINNET_API_KEY,
   WALLET_PRIV_KEY,
-} from "../__tests__/constants";
-import { OpenSeaSDK } from "../sdk";
-import { Network } from "../types";
+} from "../constants";
 
 export const TOKEN_ADDRESS = process.env.SELL_ORDER_CONTRACT_ADDRESS;
 export const TOKEN_ID = process.env.SELL_ORDER_TOKEN_ID;
 export const LISTING_AMOUNT = process.env.LISTING_AMOUNT;
 export const ETH_TO_WRAP = process.env.ETH_TO_WRAP;
 
-const webProvider = new Web3.providers.HttpProvider(
-  `https://eth-mainnet.g.alchemy.com/v2/${ALCHEMY_API_KEY}`
-);
-
 const rpcProvider = new ethers.providers.JsonRpcProvider(
   `https://eth-mainnet.g.alchemy.com/v2/${ALCHEMY_API_KEY}`
 );
 
-const wallet = new ethers.Wallet(WALLET_PRIV_KEY ?? "", rpcProvider);
+if (!WALLET_PRIV_KEY) {
+  throw new Error("WALLET_PRIV_KEY must be set for integration tests");
+}
```

Comment:
There is probably more env variables we should check here but this is still better than nothing

Reply:
will add the rest